Search.java

```diff
@@ -0,0 +1,59 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+class Search {
+
+    public static void main(String[] args) {
+        boolean case_sensitivity = true;
+        Scanner file_scanner;
+        String input_file = args[1];
```

Comment:
Input parsing is a little cluttered with variable assignment occuring outside of and within if-statement.
A comment to clarify that this assignment is made with the assumption of case sensitivity being active would perhaps be nice!

I would personally prefer if this assignment was made in the else-statement after the if-statement below, just to further indicate how this input parsing is mutually exclusive from the one in the if-statement!